editor/scene_tree_dock.cpp

```diff
@@ -986,9 +986,10 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 				} else {
 					new_node = Object::cast_to<Node>(ClassDB::instance(selected_favorite_root));
 				}
+
 				if (!new_node) {
-					ERR_EXPLAIN("Creating root from favorite '" + selected_favorite_root + "' failed. Creating 'Node' instead.");
 					new_node = memnew(Node);
+					ERR_FAIL_COND_MSG(!new_node, "Creating root from favorite '" + selected_favorite_root + "' failed. Creating 'Node' instead.");
```

Comment:
This adds a bug, the previous code did not fail. It set an error and then nothing was showing it (no `ERR_FAIL`), so what was intended is likely an `ERR_PRINTS`.